submarine-cloud-v2/controller.go

```diff
@@ -1134,6 +1136,17 @@ func (c *Controller) syncHandler(workqueueItem WorkQueueItem) error {
 		if err != nil {
 			return err
 		}
+
+		// Port-forwarding
+		// TODO:
+		//   (1) multi-tenant port-forwarding
+		//   (2) Basic operations: on/off/modify (change port)
+		//   (3) in-cluster
+		if action == ADD {
+			if !incluster {
+				k8sutil.ServicePortForwardPort(context.TODO(), newNamespace, "traefik", 32080, 80, color.FgGreen)
+			}
+		}
 	} else { // Case: DELETE
```

Comment:
You do not remove the port forward in delete event. So if the submarine custom resource is deleted and re-create again, some bug will happen.

Reply:
I have fixed it.